src/main/pg/gyrodev.c

```diff
@@ -71,20 +71,32 @@ PG_REGISTER_ARRAY_WITH_RESET_FN(gyroDeviceConfig_t, MAX_GYRODEV_COUNT, gyroDevic
 void pgResetFn_gyroDeviceConfig(gyroDeviceConfig_t *devconf)
 {
     devconf[0].index = 0;
+    sensorAlignment_t customAlignment1 = {{ 0, 0, 0 }}; // CUSTOM_ALIGN_CW0_DEG
+#ifdef GYRO_1_CUSTOM_ALIGN
```

Comment:
Can we leave these in `common_post.h` please?
It will be much easier to identify / eliminate all of these target specific defines if we leave them in there once we have moved to Unified Targets.

Reply:
The problem is that the lack of a custom alignment definition in the target drives the logic to calculate the axis alignments from the sensor alignment. So defaulting to `CUSTOM_ALIGN_CW0_DEG` in `common_post.h` means we can't differentiate between the that state or none defined. I considered using a `CUSTOM_ALIGN_NONE` instead but the problem is that this is not a simple define. It's a macro containing the axis values and testing for that instead of not defined should be troublesome.

Do we maybe just want to start a documentation section in one of the includes for target defines like this that may be infrequently used?

Reply:
Eh, that's fugly.

But yes, I guess adding a comment to this respect in place of the original defines in `common_post.h` should at least give us a pointer to rip this out when we get rid of the compile time defaults.